mission_list.md

```diff
@@ -0,0 +1,60 @@
+# Mission List
+
+Quote form the [constitution](https://github.com/polkadot-tooling-collective/constitution):
+
+>The Mission List contains all projects and services that are maintained by PoToC.
+>
+>It is divided into two sections, solely to keep the requirements of each section clearly defined; Core and dApp.
+>It can be updated through an on-chain 2/3-majority rank-weighted vote across all Members.
+>
+>Some obvious cornerstones that must be upheld by all projects are:
+>
+>- Must be FOSS (Free and Open-Source Software).
+>- Source code must be accessible without a login requirement.
+>- All members of PoToC must have permission to propose changes and raise issues.
+>- The Mission List will be defined as part of the initial seeding.
+>
+>Core Tooling
+>
+>Generally, all tools that are used by The Fellowship to craft, test, and validate updates fall into this category. But only the tools that are predominantly developed for this cause shall be maintained by PoToC. For example, the Rust language is not predominantly developed for Polkadot; hence, it does not fit here.
+>
+>dApp Tooling
+>
+>This category contains all tools and libraries that are used to build on Polkadot and are predominantly developed for this cause.
+
+(This is my personal opinion on a 
+
+## DApp Tooling
+
+- [Polkadot-JS](https://github.com/polkadot-js/apps)
+- [PAPI](https://www.npmjs.com/package/@polkadot-api/substrate-client)
+- [Chopsticks](https://github.com/AcalaNetwork/chopsticks)
+- [Zombienet](https://github.com/paritytech/zombienet) + [v2](https://github.com/paritytech/zombienet-sdk)
+- [Substrate Connect](https://github.com/paritytech/substrate-connect)
+- [Scale TS](https://npmjs.com/package/scale-ts)
+
+## Core Tooling
+
+- [SubXt](https://github.com/paritytech/subxt)
+- [Subway](https://github.com/AcalaNetwork/subway)
+- [Zepter](https://github.com/ggwpez/zepter)
+- [Substrate Weight Compare](https://github.com/ggwpez/substrate-weight-compare)
+- [OpenGov CLI](https://github.com/joepetrowski/opengov-cli)
+- [Sidecar](https://github.com/paritytech/substrate-api-sidecar)
+- [scale-*](https://github.com/paritytech/scale-decode), [more](https://github.com/polkadot-tooling-collective/collective/blob/dd96bc056dfa47f0bc6a885c7afd25ed78345a11/join_request/jameswilson.yml#L26)
+- [srtool](https://github.com/paritytech/srtool) (maintainer did not apply yet)
+- [subwasm](https://github.com/chevdor/subwasm) (ditto)
+- [virto SDK/libwallet](https://github.com/virto-network/virto-sdk/tree/main/libwallet)
+- [virto SDK/scales](https://github.com/virto-network/virto-sdk/tree/main/scales)
+
+## Undecided
+
+- [virto SDK/sube](https://github.com/virto-network/virto-sdk/tree/main/sube)
+- [polkadot-JS extension](https://github.com/polkadot-js/extension)
+- [cypress polkadot wallet](https://github.com/ChainSafe/cypress-polkadot-wallet)
+- [Zondax Ledger](https://github.com/zondax/ledger-polkadot)
```

Comment:
This is a full-fledged app. IMHO it is therefore not a *Tool* but an end-user project.

But i think we should add `https://github.com/zondax/ledger-substrate-js`, did not see it before.

Reply:
Agreed that a full-fledged app should not be part of the Tooling collective,

Reply:
`cypress polkadot wallet` is a plugin library for the Cypress testing framework. It's a tool for devs to test their Dapp. Maybe the naming was not clear?

Reply:
I agree as well. I thought differently when applying though, since it's not a full product, it requires other companion projects to work. It's in a similar position as polkadot-js/extension. Just a keyring that holds keys, not a tool per-se.

ledger-substrate-js could be a great fit👍🏻

Reply:
Added `ledger-substrate-js`.